rust/core-lib/src/rpc.rs

```diff
@@ -322,6 +322,12 @@ pub struct MouseAction {
     pub click_count: Option<u64>,
 }
 
+#[derive(Serialize, Deserialize, PartialEq, Eq, Debug)]
+pub struct Position {
```

Comment:
I'd remove this, and just include `line` and `col` params like we do with the gesture RPC, if only to keep the protocol _somewhat_ consistent.

(in fact, it might make sense for us to be just adding a `hover` `GestureType`?)

Reply:
I added this Position struct since the whole parameter i.e. line/col is optional, it uses caret position as a fallback (this makes more sense for terminal clients). However, adding this as a gesture type is a good possibility but how do terminal client invoke this then?

Reply:
Okay, good point.